getting_started/step_by_step/your_first_game.rst

```diff
@@ -68,21 +67,21 @@ node to the scene.
 
 .. image:: img/add_node.png
 
-With ``Area2D`` we can detect other objects that overlap or run into the player.
-Change its name to ``Player``. This is the scene's "root" or top-level node.
-We can add additional nodes to the player to add functionality.
+With ``Area2D`` we can detect objects that overlap or run into the player.
+Change its name to ``Player`` by clicking on the node's name.
+This is the scene's root node. We can add additional nodes to the player to add functionality.
 
 Before we add any children to the ``Player`` node, we want to make sure we don't
-accidentally move or resize them by clicking on them. Select the player node and
-click the icon next to the lock - its tooltip says "Makes sure the objects children
+accidentally move or resize them by clicking on them. Select the node and
+click the icon to the right of the lock; its tooltip says "Makes sure the object's children
 are not selectable."
 
 .. image:: img/lock_children.png
 
-Save the scene (click Scene -> Save, or press ``Control+S`` on Windows/Linux or ``Command+S`` on Mac).
+Save the scene. Click Scene -> Save, or press ``Ctrl+S`` on Windows/Linux or ``Command+S`` on Mac.
 
-.. note:: In this project, we will be following the Godot Engine naming
-          conventions. Classes (Nodes) use ``CapWords``, variables and
+.. note:: For this project, we will be following the Godot naming
+          conventions. Classes (nodes) use ``CamelCase``, variables and
           functions use ``snake_case``, and constants use ``ALL_CAPS``.
```

Comment:
This is incorrect. "CamelCase" generally refers to the capitalization style where the initial letter is lowercase: camelCase.  We could more accurately use "PascalCase" here.

Reply:
From Wikipedia: "Camel case is the practice of writing compound words or phrases such that each word or abbreviation in the middle of the phrase begins with a capital letter, with no intervening spaces or punctuation. Common examples include "iPhone ", "eBay", "FedEx", "DreamWorks", "HarperCollins", "iCarly", "WordWorld", and "WordGirl"."

The rules for punctuation are inconsistent; I have never heard of PascalCase before, despite writing about games for the past four years, and I doubt anybody getting started in game development for the first time will have either.

Reply:
I can confirm that when it comes to code camel case will most likely mean lowercase first. And pascal case or a million others for class names, which start with a capital letter. 

> I have never heard of PascalCase before, despite writing about games for the past four years, and I doubt anybody getting started in game development for the first time will have either.

People who don't have programming experience likely won't have heard of camel case, snake case, etc. either. The expression doesn't matter too much but I'd bet with the C# devs coming to the engine if you leave camel case you'll see PRs pop up to change it quickly.

Reply:
Camel Case is a general term and technically is valid for both "camelCase" and "CamelCase". But in the programming world it's common to differentiate the two with "PascalCase" used when it starts with capital and "camelCase" otherwise.